createDoctorWhoDB.sql

```diff
@@ -0,0 +1,135 @@
+CREATE TABLE [tblAuthor]
+(
+ [AuthorId]   int NOT NULL ,
```

Comment:
Integer PKs should be better auto incremented. So when you insert a new author, you don't have to provide the ID.

Reply:
this applies to all the other tables in this file